src/Layouts/Partials/FillItem.vala

```diff
@@ -243,12 +314,7 @@ public class Akira.Layouts.Partials.FillItem : Gtk.Grid {
     private void on_color_changed () {
         var selected_color = color_chooser_widget.rgba;
 
-        color = "#%02X%02X%02X".printf (
-            (int) (selected_color.red * 255),
-            (int) (selected_color.green * 255),
-            (int) (selected_color.blue * 255));
-
-        alpha = selected_color.alpha;
+        color = selected_color.to_string ();
```

Comment:
No need to declare the `selected_color` variable if it's used only once.
You can directly do:
`color = color_chooser_widget.rgba.to_string ();`